SIT.Manager.Avalonia/Views/MainView.axaml

```diff
@@ -25,11 +25,14 @@
 			</ui:TeachingTip.IconSource>
 		</ui:TeachingTip>
 
-		<StackPanel Name="InfoBarStackPanel" Grid.Row="0" x:FieldModifier="Public" IsVisible="True">
-			<ui:InfoBar Name="UpdateInfoBar" IsOpen="False">
+		<StackPanel Name="InfoBarStackPanel" Grid.Row="0" x:FieldModifier="Public">
+			<ui:InfoBar Name="UpdateInfoBar" IsOpen="{Binding UpdateAvailable}" MaxHeight="45" Margin="0 0 0 10">
```

Comment:
Feel like the the margin of 10 on the bottom of this bar makes it look the most weird. Reckon it should just be `Margin="0"` or could just have the whole Margin bit deleted.

Reply:
Thats my bad. I was trying to add a margin to correct the alignment of things a little but the margin didn't seem to do anything for it. The entire infobar is handled weirdly and i'm kinda fighting with it